pages/api/candidateApiClient.module.ts

```diff
@@ -1,36 +1,24 @@
 ﻿import fetch from "node-fetch";
```

Comment:
Would you mind moving this out of the `pages` directory?
Its not really a page, so fine to just have a `/api` directory at the root level.

Reply:
yep, no problem, it was just what they did in the Nextjs documentation so I blindly followed that. changed now 👍